src/Catalog/Entry.php

```diff
@@ -244,7 +245,7 @@ public function isFuzzy()
      */
     public function isPlural()
     {
-        return $this->getMsgIdPlural() !== null || $this->getMsgStrPlurals() !== null;
+        return $this->getMsgIdPlural() !== null || count($this->getMsgStrPlurals()) > 0;
```

Comment:
Just a suggestion: Better use `empty` instead of `count` to check if there are any elements in the array, as it is faster.

Reply:
I don't have a strong opinion about this, but I tend to use `count` over `empty` to check for no items in an array.

Reply:
Ok, but as after this change the variable you're testing is always an array, that PHP magic will never apply :)

Reply:
What do you mean with "that PHP magic will never apply"?

Reply:
We can get rid off of that `$this->getMsgIdPlural() !== null`

Reply:
Are you sure? I thought it is there intentionally, because it checks the ID rather then the strings.

These should be probably in sync, but I am not sure what the specification says and it is not checked anywhere in the object, that when there are pluralization strings, then the ID has to be also present, that is why I left it there.